packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -363,18 +515,37 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
             id,
             minWidth,
             isOpen,
-            focusedIndex,
+            focusedIndex: undefined,
             enableVirtualization,
             filteredOptions,
             getMenuItem,
+            multiple,
           }}
         >
           <ComboBoxInputContainer disabled={inputProps?.disabled} {...rest}>
-            <ComboBoxInput
-              value={inputValue}
-              {...inputProps}
-              onChange={handleOnInput}
-            />
+            <>
+              <ComboBoxInput
+                style={{
+                  paddingLeft: tagContainerWidth + 18,
```

Comment:
Add dynamic padding change and dynamic height change (for custom rendering if users don't handle overflow)